source/_posts/angular-analytics.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Google analytics with Tag manager and Angular
+date: 2020-10-25 13:14:41
+author: Abhishek Rana
+category:
+  - ["Web"]
+tags:
```

Comment:
NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING
Please choose the keyword related to your blog and put it in your tag section as well as in content also